bindata/manifests/webhook/001-service.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: {{.Namespace}}
   annotations:
     {{- if eq .ClusterType "openshift" }}
-    service.alpha.openshift.io/serving-cert-secret-name: network-resources-injector-secret
+    service.alpha.openshift.io/serving-cert-secret-name: {{.InjectorWebhookSecretName}}
```

Comment:
nit: can you refer to resource injector as `{r,R}esourceInjector` or `{n,N}etworkResourceInjector` througout ? 
IMO injector its too general